admin_organization.go

```diff
@@ -0,0 +1,110 @@
+package tfe
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+)
+
+// Compile-time proof of interface implementation.
+var _ AdminOrganizations = (*adminOrganizations)(nil)
+
+// AdminOrganizations describes all of the admin organization related methods that the Terraform
+// Enterprise API supports. Note that admin settings are only available in Terraform Enterprise.
+//
+// TFE API docs: https://www.terraform.io/docs/cloud/api/admin/organizations.html
+type AdminOrganizations interface {
+	// Read attributes of an existing organization via admin API.
+	Read(ctx context.Context, organization string) (*AdminOrganization, error)
+
+	// Update attributes of an existing organization via admin API.
+	Update(ctx context.Context, organization string, options AdminOrganizationUpdateOptions) (*AdminOrganization, error)
+
+	// Delete an organization by its name via admin API
+	Delete(ctx context.Context, organization string) error
+}
+
+// adminOrganizations implements AdminOrganizations.
+type adminOrganizations struct {
+	client *Client
+}
+
+// AdminOrganization represents a Terraform Enterprise organization returned from the Admin API.
+type AdminOrganization struct {
+	Name string `jsonapi:"primary,organizations"`
+
+	AccessBetaTools                  bool   `jsonapi:"attr,access-beta-tools"`
+	ExternalID                       string `jsonapi:"attr,external-id"`
+	GlobalModuleSharing              bool   `jsonapi:"attr,global-module-sharing"`
+	IsDisabled                       bool   `jsonapi:"attr,is-disabled"`
+	NotificationEmail                string `jsonapi:"attr,notification-email"`
+	SsoEnabled                       bool   `jsonapi:"attr,sso-enabled"`
+	TerraformBuildWorkerApplyTimeout string `jsonapi:"attr,terraform-build-worker-apply-timeout"`
+	TerraformBuildWorkerPlanTimeout  string `jsonapi:"attr,terraform-build-worker-plan-timeout"`
+	TerraformWorkerSudoEnabled       bool   `jsonapi:"attr,terraform-worker-sudo-enabled"`
+}
+
+// AdminOrganizationUpdateOptions represents the admin options for updating an organization.
+// https://www.terraform.io/docs/cloud/api/admin/organizations.html#request-body
+type AdminOrganizationUpdateOptions struct {
+	AccessBetaTools                  *bool   `jsonapi:"attr,access-beta-tools,omitempty"`
+	GlobalModuleSharing              *bool   `jsonapi:"attr,global-module-sharing,omitempty"`
+	IsDisabled                       *bool   `jsonapi:"attr,is-disabled,omitempty"`
+	TerraformBuildWorkerApplyTimeout *string `jsonapi:"attr,terraform-build-worker-apply-timeout,omitempty"`
+	TerraformBuildWorkerPlanTimeout  *string `jsonapi:"attr,terraform-build-worker-plan-timeout,omitempty"`
+}
```

Comment:
I think this is missing `terraform-worker-sudo-enabled`, right?

Reply:
I think theres a GitHub UI bug, because the [field is there](https://github.com/hashicorp/go-tfe/pull/180/files#diff-7912c8c3b0c986d17c6735dd82ae875a2640e531ac645a12faec3d22be2c12b5R43).

Reply:
This option still needs adding for this struct!

Reply:
Added